code/modules/power/lighting/light_alarm.dm

```diff
@@ -0,0 +1,35 @@
+/obj/machinery/light/alarm //A light which turns on only during Code Red or above
+	name = "alarm light"
+	icon = 'icons/obj/lighting.dmi'
+	desc = "A light signalling if there is a serious situation on the station."
+	base_state = "alarmlight"
+	icon_state = "alarmlight"
+	max_integrity = 180
+	brightness = 6
+	bulb_emergency_colour = "#f1bc0d"
+	bulb_colour = "#e61313"
+	nightshift_allowed = FALSE
+	no_low_power = TRUE
+	light_type = /obj/item/light/bulb
+	fitting = "bulb"
+
+/obj/machinery/light/alarm/Initialize(mapload) //Makes it register security level change
+	. = ..()
+	RegisterSignal(SSsecurity_level, COMSIG_SECURITY_LEVEL_CHANGED, PROC_REF(emergency))
+
+/obj/machinery/light/attackby(obj/item/tool, mob/living/user, params) //If people decide to stick a new lightbulb in it, it will not turn on by itself
```

Comment:
supposed to be on alarm

Reply:
Huh?

Reply:
`/obj/machinery/light/attackby`, you mean `light/alarm/attackby`

Reply:
> `/obj/machinery/light/attackby`, you mean `light/alarm/attackby`

Oooh this, for some reason when I changed it to "alarm" I was unable to change it's lightbulb